cmd/kubeadm/app/componentconfigs/kubeproxy.go

```diff
@@ -49,7 +51,14 @@ var kubeProxyHandler = handler{
 }
 
 func kubeProxyConfigFromCluster(h *handler, clientset clientset.Interface, _ *kubeadmapi.ClusterConfiguration) (kubeadmapi.ComponentConfig, error) {
-	return h.fromConfigMap(clientset, kubeadmconstants.KubeProxyConfigMap, kubeadmconstants.KubeProxyConfigMapKey, false)
+	configMapName := kubeadmconstants.KubeProxyConfigMap
+	klog.V(1).Infof("attempting to download the KubeProxyConfiguration from ConfigMap %q", configMapName)
```

Comment:
late review 🤣

We usually include the name for configmap in log like:  `ConfigMap "kubeadm-config" in the "kube-system" Namespace`.